packages/cryptoplease_ui/pubspec.yaml

```diff
@@ -4,31 +4,17 @@ version: 0.0.1
 publish_to: none
 
 environment:
-  sdk: ">=2.17.0 <3.0.0"
-  flutter: ">=3.0.0"
+  sdk: '>=2.17.0 <3.0.0'
+  flutter: '>=3.0.0'
 
 dependencies:
   dfunc: ^0.7.2
   flutter:
     sdk: flutter
+  google_fonts: ^3.0.1
```

Comment:
It's better not to use this package, we don't need it to download the font over internet. Just use the same technique with fonts in assets.